[2024-12-30T20:27:40.782+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-30T20:27:40.796+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_kafka_producer.fetch_and_publish_weather scheduled__2024-12-30T20:27:35+00:00 [queued]>
[2024-12-30T20:27:40.802+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_kafka_producer.fetch_and_publish_weather scheduled__2024-12-30T20:27:35+00:00 [queued]>
[2024-12-30T20:27:40.802+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-30T20:27:40.813+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_and_publish_weather> on 2024-12-30 20:27:35+00:00
[2024-12-30T20:27:40.820+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2439) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-30T20:27:40.821+0000] {standard_task_runner.py:72} INFO - Started process 2441 to run task
[2024-12-30T20:27:40.822+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_kafka_producer', 'fetch_and_publish_weather', 'scheduled__2024-12-30T20:27:35+00:00', '--job-id', '464', '--raw', '--subdir', 'DAGS_FOLDER/weather_kafka_producer.py', '--cfg-path', '/tmp/tmpltc5ssk8']
[2024-12-30T20:27:40.823+0000] {standard_task_runner.py:105} INFO - Job 464: Subtask fetch_and_publish_weather
[2024-12-30T20:27:40.860+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_kafka_producer.fetch_and_publish_weather scheduled__2024-12-30T20:27:35+00:00 [running]> on host f4979116b9f3
[2024-12-30T20:27:40.928+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_kafka_producer' AIRFLOW_CTX_TASK_ID='fetch_and_publish_weather' AIRFLOW_CTX_EXECUTION_DATE='2024-12-30T20:27:35+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-30T20:27:35+00:00'
[2024-12-30T20:27:40.930+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-30T20:27:41.139+0000] {logging_mixin.py:190} INFO - Error publishing to Kafka: name 'city_id' is not defined
[2024-12-30T20:27:41.140+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-30T20:27:41.149+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-30T20:27:41.150+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_kafka_producer, task_id=fetch_and_publish_weather, run_id=scheduled__2024-12-30T20:27:35+00:00, execution_date=20241230T202735, start_date=20241230T202740, end_date=20241230T202741
[2024-12-30T20:27:41.196+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-30T20:27:41.212+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-30T20:27:41.215+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
